src/math/bigintops.c

```diff
@@ -817,6 +817,32 @@ void MVM_bigint_from_str(MVMThreadContext *tc, MVMObject *a, const char *buf) {
     }
 }
 
+MVMObject * MVM_bigint_from_bigint(MVMThreadContext *tc, MVMObject *result_type, MVMObject *a) {
+    MVMP6bigintBody *a_body  = get_bigint_body(tc, a);
```

Comment:
This returns an interior pointer into `a`, so this assignment should be moved until after the `MVMROOT` also.

Reply:
Fixed